app/models/conversion_host.rb

```diff
@@ -241,20 +241,25 @@ def resource_supports_conversion_host
   end
 
   # Find the credentials for the associated resource. By default it will
-  # look for a v2v auth type. If that is not found, it will look for the
-  # authentication associated with the resource using ssh_keypair or default,
-  # in that order, as the authtype.
+  # look for a v2v auth type if no argument is passed in. If that is not found,
+  # it will look for the first associated authentication. If one still isn't
+  # found, then it will look for the authentication associated with the resource
+  # using the 'ssh_keypair' auth type, and finally 'default'.
   #
-  def find_credentials(msg = nil)
-    authentication = authentication_type('v2v') ||
-      resource.authentication_type('ssh_keypair') ||
-      resource.authentication_type('default')
+  def find_credentials(auth_type = 'v2v')
+    authentication = authentication_type(auth_type) ||
+      authentication_type('ssh_keypair')            ||
+      authentication_type('default')
```

Comment:
Wait I thought we just said v2v was the only auth type that could be added?

Reply:
Ok, updated.